php/Blockonomics.php

```diff
@@ -129,12 +129,13 @@ private function get($url, $api_key = '')
         }
     }
 
-    private function post($url, $api_key = '', $body = '', $type = '')
```

Comment:
Let us not expicitly use the default timeout of 5 sec, since later wp_remote_post could change default. 
When using default don't set the parameter or set to null